recipe/gen_patch_json.py

```diff
@@ -545,6 +545,15 @@ def _gen_new_index(repodata, subdir):
                     _relax_exact(fn, record, llvm, max_pin="x.x")
                 else:
                     _relax_exact(fn, record, llvm, max_pin="x.x.x")
+
+        if record_name in llvm_pkgs:
```

Comment:
I don't see the def of `llvm_pkgs`

Reply:
It's defined  a few lines above

Reply:
ahhhhh I thought it would be in this diff!